easybuild/easyblocks/n/neuron.py

```diff
@@ -57,6 +59,7 @@ def extra_options():
 
         extra_vars = {
             'paranrn': [True, "Enable support for distributed simulations.", CUSTOM],
+            'enable_tests': [True, "Enable extensive testing in the sanity check step.", CUSTOM],
```

Comment:
@damianam we could move this to the `test_step` method, but then we need to do some extra effort to make sure that the environment is such that the NEURON that was built can be used; in the `sanity_check_step` we simply rely on (a temporary version of) the generated module to handle that for us

So, the test being done in the sanity check step somewhat validates the installation + the module that will be generated...